xbmc/cores/VideoPlayer/DVDCodecs/Video/VAAPI.cpp

```diff
@@ -1209,7 +1209,7 @@ void CDecoder::Register(IVaapiWinSystem *winSystem, bool deepColor)
   CServiceBroker::GetSettings().GetSetting(CSettings::SETTING_VIDEOPLAYER_USEVAAPIMPEG2)->SetVisible(true);
   CServiceBroker::GetSettings().GetSetting(CSettings::SETTING_VIDEOPLAYER_USEVAAPIVP8)->SetVisible(true);
   CServiceBroker::GetSettings().GetSetting(CSettings::SETTING_VIDEOPLAYER_USEVAAPIVP9)->SetVisible(true);
-  CServiceBroker::GetSettings().GetSetting(CSettings::SETTING_VIDEOPLAYER_USEVAAPIHEVC)->SetVisible(true);
+  CServiceBroker::GetSettings().GetSetting("videoplayer.usevaapihevc")->SetVisible(true);
```

Comment:
Moving the string out of generic settings was a good idea, but could you use a #define in the VAAPI file to avoid duplicating the string?

Reply:
I'd rather use a const within vaapi as #defines do not honor namespaces etc.

Reply:
sure, const is a better idea

Reply:
@FernetMenta what would you like to see done?